CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [UNRELEASED] - 2025-01-08
+ - Format [blue](https://github.com/JuliaDiff/BlueStyle)
```

Comment:
Could you file that under "Changed", maybe as

```
- Reformat sources with [blue](https://github.com/JuliaDiff/BlueStyle) style
```

I would like to use `Added`, `Changed`, `Removed`, and `Fixed` sections like in https://github.com/cbrnr/mnelab/blob/main/CHANGELOG.md, could you please add the missing `Added` for the previous release?